test/framework/src/main/java/org/elasticsearch/test/hamcrest/ElasticsearchAssertions.java

```diff
@@ -623,6 +624,14 @@ private static BytesReference serialize(Version version, Streamable streamable)
         return output.bytes();
     }
 
+    private static BytesReference serialize(Version version, Writeable writeable) throws IOException {
```

Comment:
Is this method used anywhere? `assertVersionSerializable` below operates on `Streamable`.